src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -4,54 +4,33 @@
 
 use ArrayAccess;
 use Faker\Generator as Faker;
+use Illuminate\Database\Eloquent\Factory\StateManager;
 use Symfony\Component\Finder\Finder;
 
 class Factory implements ArrayAccess
 {
-    /**
-     * The model definitions in the container.
-     *
-     * @var array
-     */
-    protected $definitions = [];
-
-    /**
-     * The registered model states.
-     *
-     * @var array
-     */
-    protected $states = [];
-
-    /**
-     * The registered after making callbacks.
-     *
-     * @var array
-     */
-    protected $afterMaking = [];
-
-    /**
-     * The registered after creating callbacks.
-     *
-     * @var array
-     */
-    protected $afterCreating = [];
-
     /**
      * The Faker instance for the builder.
      *
      * @var \Faker\Generator
      */
     protected $faker;
 
+    /**
+     * @var StateManager
+     */
+    protected $stateManager;
+
     /**
      * Create a new factory instance.
      *
-     * @param  \Faker\Generator  $faker
-     * @return void
+     * @param  \Faker\Generator $faker
+     * @param StateManager $stateManager
```

Comment:
Use FQN in DocBlocks and two spaces before and after types in `@param` annotations.

Ps.: I'm gonna skip all the other places in this PR where this is the case so it's best that you go through it yourself.